django/utils/decorators.py

```diff
@@ -57,7 +67,7 @@ def bound_func(*args2, **kwargs2):
         # want to copy those. We don't have access to bound_func in this scope,
         # but we can cheat by using it on a dummy function.
 
-        @decorator
+        @decorate
         def dummy(*args, **kwargs):
             pass
         update_wrapper(_wrapper, dummy)
```

Comment:
citing the inlines comments ``django.utils.decorators:66``:
```python
        # In case 'decorator' adds attributes to the function it decorates, we
        # want to copy those. We don't have access to bound_func in this scope,
        # but we can cheat by using it on a dummy function.
```
This is the justification for the ``decorate(dummy)`` thing, to be sincere I'm not sure about the necessity of that since it is an hack that implies some sort of side effect/bug probably of python itself, it would have been great if it was referred to.
Anyway I completed the argument  tests for ``method_decorator()``.